kodilla-patterns2/src/main/java/com/kodilla/kodillapatterns2/facade/api/WatcherFacade.java

```diff
@@ -0,0 +1,20 @@
+package com.kodilla.kodillapatterns2.facade.api;
+
+import org.aspectj.lang.annotation.Aspect;
+import org.aspectj.lang.annotation.Before;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.stereotype.Component;
+
+@Aspect
+@Component
+public class WatcherFacade {
```

Comment:
Nazwa klasy sugeruje, że jest tutaj wykorzystany wzorzec fasada. To jest aspekt do fasady, fasady ordera - więc lepiej byłoby nazwać np. `OrderWatcher`.